configs/coderello_docs.json

```diff
@@ -0,0 +1,26 @@
+{
+  "index_name": "coderello_docs",
```

Comment:
Changing the name of the configuration will require additional steps on our side. The reason why you want to change the name of it are legit, we will update it once deploying this PR. You can disregard the circle ci check.

Reply:
Thanks.